src/main/java/org/broadinstitute/consent/http/resources/UserResource.java

```diff
@@ -148,6 +137,31 @@ public Response addRoleToUser(@Auth AuthUser authUser, @PathParam("userId") Inte
         }
     }
 
+    @DELETE
+    @Path("/{userId}/{roleId}")
+    @Produces("application/json")
+    @RolesAllowed({ADMIN})
+    public Response deleteRoleFromUser(@Auth AuthUser authUser, @PathParam("userId") Integer userId, @PathParam("roleId") Integer roleId) {
+        try {
+            User user = userService.findUserById(userId);
+            if (!UserRoles.isValidNonDACRoleId(roleId)) {
+                throw new BadRequestException("Role Id must be an Integer representing an allowed role (3 to 7)");
```

Comment:
Let's simplify this message - we clearly spell out the requirements in Swagger.
```suggestion
                throw new BadRequestException("Invalid Role Id");
```